styles/main.css

```diff
@@ -1,14 +1,58 @@
 @CHARSET "UTF-8";
-body {background-color : yellow;}
-h1, h2 {color : red; background-image: url("../images/gradient.png"); background-repeat: repeat-x; font-family : Lora, serif}
-p {font-family : Roboto, serif}
-hyperlink {color : green; text-decoration : none}
-nav hyperlink {color : black}
-.byline {color: rgba(  0,122, 88,0.5)}
-#page-copyright {color : indigo; text-transform : uppercase; letter-spacing : 3px;}
-.hakiu {background-color : black}
-.bio {text-indent: 3em;}
-footer {text-align: center; font-size : 12px;}
-.byline {font-size : 70%;}
-header {list-style-type : none; }

```

Comment:
Incorrect selector.  `list-style-type` only applies to lists so you need something like `header ul` as your selector